packages/ui/src/components/primitives/ChainIcon.tsx

```diff
@@ -19,21 +20,30 @@ const ChainIcon: FC<Props> = ({
   square = true
 }) => {
   const client = useRelayClient()
-  const icon = chainId
-    ? client?.chains?.find((chain) => chain.id === chainId)?.icon?.light
+  const chain = chainId
+    ? client?.chains?.find((chain) => chain.id === chainId)
     : null
 
-  const iconUrl = square ? icon?.replace('/icons/', '/icons/square/') : icon
+  const icon = chain
+    ? isDarkMode()
```

Comment:
I would double check if we even have dark mode assets, for the most part we've only been uploading/using light mode icons.

Reply:
Yeah good point. I've verified them all and we fallback to the light mode icon in cases where there isn't a specific dark mode icon